managers/muse.py

```diff
@@ -121,12 +121,24 @@ def constructWebpubManifest(self):
                 pdfManifest.closeSection()
 
             if pdfItem:
-                pdfManifest.addChapter(
-                    '{}{}'.format(
-                        self.MUSE_ROOT, pdfItem.parent.get('href')
-                    ),
-                    titleItem.span.a.string,
-                )
+                #skipCover indicates whether to skip the intersitital cover page by adding a query parameter
+                skipCover = False
```

Comment:
I think this needs to be set outside of the `for` loop otherwise it will always evaluate to `False` here.

Reply:
Agreed. I moved it outside the for loop.